client/components/marketing-survey/cancel-jetpack-form/jetpack-cancellation-offer-accepted.tsx

```diff
@@ -0,0 +1,38 @@
+import { useTranslate } from 'i18n-calypso';
+import * as React from 'react';
+import cloudIcon from 'calypso/assets/images/jetpack/cloud-icon.svg';
+import FormattedHeader from 'calypso/components/formatted-header';
+import JetpackDecorativeCard from 'calypso/components/jetpack-decorative-card';
+
+interface Props {
+	siteId: number;
+	percentDiscount: number;
+	productName: string;
+}
+
+const JetpackCancellationOfferAccepted: React.FC< Props > = ( props ) => {
+	const translate = useTranslate();
+	const { percentDiscount, productName } = props;
+
+	return (
+		<>
+			<JetpackDecorativeCard iconPath={ cloudIcon } />
+			<FormattedHeader
+				headerText={ translate( 'Thanks for sticking with Jetpack!' ) }
+				subHeaderText={ translate(
+					'We’re happy you’ve chosen Jetpack to level-up your site. Your %(percentDiscount)d%% discount for %(productName)s will be applied next time you are billed.',
```

Comment:
`%(percentDiscount)d%%` might be difficult for translators to read into. Maybe it'd be safer to inject the percentage amount with the symbol in the string. Or at least add a comment for translators.

Reply:
Good thought there 👍